PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -252,7 +255,54 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         esEffSigmaRR = Var("full5x5_showerShapeVariables().effSigmaRR()", float, doc="preshower sigmaRR"),
         esEnergyOverRawE = Var("superCluster().preshowerEnergy()/superCluster().rawEnergy()", float, doc="ratio of preshower energy to raw supercluster energy"),
         haloTaggerMVAVal = Var("haloTaggerMVAVal()",float,doc="Value of MVA based BDT based  beam halo tagger in the Ecal endcap (valid for pT > 200 GeV)",precision=8),
-    )
+                
+)
+
+photonTable.variables = cms.PSet(
+    _phoVarsBase
+)
+
+photonExtraTable = photonTable.clone()
+
+_phoVarsExtra = cms.PSet(
+    r9Frac = Var("r9()",float,doc="Fractional R9 of the supercluster",precision=10),
+    superclusterEta = Var("superCluster().eta()",float,doc="supercluster eta",precision=10),
```

Comment:
This was added by https://github.com/cms-sw/cmssw/pull/43198, so it can be removed from here.